armi/physics/constants.py

```diff
@@ -22,6 +22,8 @@
 Notes
 -----
 This data structure can be updated by plugins with design-specific dpa data.
+
+:meta hide-value:
```

Comment:
What does this do? (Sorry.)

Reply:
Are you trying to hide the "Notes"?

Reply:
This explains it better than I can: 
![image](https://github.com/terrapower/armi/assets/12819067/fccd8a6e-368c-4a46-8c33-ad5f9978d542)

I basically just skips rendering the default value for the attribute.